discretize/utils/matutils.py

```diff
@@ -561,3 +563,11 @@ def __ge__(self, v):
 
     def __gt__(self, v):
         return 1 > v if self._positive else -1 > v
+
+    @property
+    def T(self):
+        return self
+
+    @property
+    def transpose(self):
```

Comment:
I think that transpose should be a function not a property:

https://github.com/scipy/scipy/blob/v0.14.0/scipy/sparse/csr.py#L108